### PR TITLE
Fixed my scrollbar mistake

### LINKED ISSUE #735: Mistake with #713
#### Nature of issue?

- [X] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Details about the bug: 
Hi ! It's me again. I unfortunately made a small CSS mistake during my work on #713, which causes a useless scroll bar to appear on the `/full/` page. I'll be correcting that quickly.

### PR DESCRIPTION
**Pull request by Vertmo**
Before your pull request is reviewed and merged, please ensure that:

* [X] there are no linting errors -- `npm run lint`
* [X] your code is in a uniquely-named feature branch and has been rebased on top of the latest master. If you're asked to make more changes, make sure you rebase onto master then too!
* [X] your pull request is descriptively named and links to an issue number, i.e. `Fixes #123`

Fixes #735

I fixed my CSS mistake (turns out it was just one line). The `full` page should look better now !

### CODE REVIEW

**Comment by catarak** [comment]
even though this fixes the styling in `/full/`, it breaks it in edit mode:
screen shot 2018-10-19 at 6 12 19 pm

**Comment by Vertmo** [comment]
Wow that's weird, I specifically tested it and it doesn't do that on my machine. Which browser are you using ?

**Comment by catarak** [comment]
using Chrome!

**Comment by Vertmo** [comment]
Weird, I tested in it too. Well I think I'll define a separate class for the full view one, and I'll do the changes only there. Give me a minute !

**Comment by catarak** [comment]
no worries!

File: client/modules/IDE/components/PreviewFrame.jsx [hunk 1]
```diff
@@ -333,7 +333,7 @@ class PreviewFrame extends React.Component {
     return (
       <iframe
         id="canvas_frame"
-        className="preview-frame"
+        className={this.props.fullView ? 'fullview-preview-frame' : 'preview-frame'}
```
Comment (catarak) [inline client/modules/IDE/components/PreviewFrame.jsx]: The pattern I've used in the web editor relies upon the [classnames](https://github.com/JedWatson/classnames) npm package, as well as [BEM](http://getbem.com/). Therefore, the code should look more like:
```js
// somewhere in render, above the return statement
let iframeClass = classNames({
  'preview-frame': true,
  'preview-frame--full-view': this.props.fullView
});

// in the return statement
<iframe
  ...
  className={iframeClass}
  ...
```
and `preview-frame--full-view` given the appropriate CSS.

**Comment by catarak** [comment]
all good! this is working for me

**Pull request merged**